streamz/dataframe/core.py

```diff
@@ -874,6 +880,22 @@ def _cb(interval, freq, source, continue_):
             last = now
 
 
+class Random(PeriodicDataFrame):
+    """Same as PeriodicDataFrame, but initialized with 3 random streams
+    (Kept for backwards compatibility and examples; otherwise not useful.)
```

Comment:
"otherwise not useful" - still useful! :)

Reply:
Ok, I'll reword that, but note that it strictly adds nothing new to PeriodicDataFrame besides the name Random. :-)  It will also retain the `freq` parameter that I think I'll rename to `cadence` in PeriodicDataFrame.

Reply:
OK